common/src/main/java/com/wynntils/wynn/model/quests/QuestScoreboardHandler.java

```diff
@@ -29,35 +23,38 @@ public void onSegmentChange(Segment newValue, ScoreboardModel.SegmentType segmen
         }
 
         StringBuilder questName = new StringBuilder();
-        StringBuilder description = new StringBuilder();
+        StringBuilder nextTask = new StringBuilder();
 
         for (String line : content) {
             if (line.startsWith("§e")) {
                 questName.append(ComponentUtils.stripFormatting(line)).append(" ");
             } else {
-                description
-                        .append(line.replaceAll(ChatFormatting.WHITE.toString(), ChatFormatting.AQUA.toString())
+                nextTask.append(line.replaceAll(ChatFormatting.WHITE.toString(), ChatFormatting.AQUA.toString())
                                 .replaceAll(ChatFormatting.GRAY.toString(), ChatFormatting.RESET.toString()))
                         .append(" ");
             }
         }
 
-        String descriptionTrimmed = description.toString().trim();
+        Optional<QuestInfo> questInfoOpt =
+                QuestManager.getQuestFromName(questName.toString().trim());
+        if (questInfoOpt.isEmpty()) {
+            WynntilsMod.warn("Cannot match quest from scoreboard to actual quest: " + questName);
+            return;
```

Comment:
Perhaps do something like 
```java
questInfo.setNextTask("Unknown");
```
if possible so the player knows that parsing has failed instead of showing an outdated task

Reply:
But if that happens we don't have a questInfo... If you're talking about what to do in case `questInfoOpt.isEmpty()`.

I think this error is unlikely enough that we don't have to worry about it. And it won't cause a crash.